lib/template.html

```diff
@@ -223,7 +223,16 @@
 {% endif %}
 
 {% if seo_page_image %}
+  {% if page.image.height && page.image.width %}
+    "image": {
+        "@type": "ImageObject",
+        "url": {{ seo_page_image | jsonify }},
```

Comment:
Based on https://developers.google.com/schemas/reference/types/MediaObject should this be `contentUrl`?